environment

```diff
@@ -18,15 +18,13 @@ DSS_SECRETS_STORE="${DSS_PLATFORM}/dss"
 
 DSS_INFRA_TAG_PROJECT="${DSS_PLATFORM}-dss"
 DSS_INFRA_TAG_SERVICE="dss"
-DSS_INFRA_TAG_OWNER="team-redwood-group@ucsc.edui"
+DSS_INFRA_TAG_OWNER="team-redwood-group@ucsc.edu"
 
-#not sure if these are needed
-OIDC_AUDIENCE=https://dss.dev.ucsc-cgp-redwood.org
-AUTH_URL=https://auth.ucsc.ucsc-cgp-redwood.org
+# dont use interpolation on the stage here, its going to change the claims that are set in the auth0_rule
+# Warning back slashes are sensitive here
```

Comment:
May want to be a little more clear what you mean here

Would be helpful to mention OIDC_AUDIENCE needs a trailing slash

Reply:
I agree wholeheartedly with this very stable genius who is making random suggestions for us in our repo